johnsonandjohnson/Bodiless-JS
# Load google font using next/fonts

## Changes


## Test Instructions


This applies changes to ensure behavior is matching between the Gatsby bundling/chunk loading and the Next.js version. Previously there was a lot of blocking time due to chunks being loaded synchronously that weren't in the Gatsby variant, making these async and only loaded on-demand helped noticeably. Additionally, in the Gatsby variant there was a `babel-plugin-lodash` optimization which wasn't also being applied in the Next.js variation so the `modularizeImport` built-in config was added to apply this there as well. 

PageSpeed report can be seen here: https://pagespeed.web.dev/analysis/https-bodiless-6shm52ygw-vtest314-ijjk-testing-vercel-app/wt48op54nh?form_factor=mobile

Related deployment here: https://bodiless-6shm52ygw-vtest314-ijjk-testing.vercel.app/

<details>

<summary>statoscope preview</summary>

![CleanShot 2023-06-12 at 18 54 13@2x](https://github.com/johnsonandjohnson/Bodiless-JS/assets/22380829/33f667b5-afb0-40a2-939a-8d31ab154614)

</details>

<details>

<summary>Lighthouse report preview</summary>

![CleanShot 2023-06-12 at 18 52 40@2x](https://github.com/johnsonandjohnson/Bodiless-JS/assets/22380829/638b2ed9-49a8-4c03-8c20-ca9c0899f07a)

</details>


## Related Issues

x-ref: [slack thread](https://bodiless-js.slack.com/archives/C04DQN4NRPD/p1685992450485259)
